compiler/src/dotty/tools/dotc/transform/CheckUnused.scala

```diff
@@ -714,12 +713,13 @@ object CheckUnused:
           if altName.exists(explicitName => selector.rename != explicitName.toTermName) then
             // if there is an explicit name, it must match
             false
-          else
+          else selData.qualTpe =:= prefix && (
```

Comment:
Let's put this on a dedicated line ;)
```suggestion
          else
            selData.qualTpe =:= prefix && (
```